### ISSUE TITLE
Determine ortools version range

### ISSUE THREAD

**Issue by arcondello** [dwavesystems/penaltymodel#54]
There was an install error for OSX 10.10.5, python2.7

```
Collecting ortools<7.0.0,>=6.6.4659 (from penaltymodel-mip<0.2.0,>=0.1.2; platform_machine != "x86" and python_version != "3.4"->dwavebinarycsp<0.1.0,>=0.0.2->dwave-ocean-sdk)
  Could not find a version that satisfies the requirement ortools<7.0.0,>=6.6.4659 (from penaltymodel-mip<0.2.0,>=0.1.2; platform_machine != "x86" and python_version != "3.4"->dwavebinarycsp<0.1.0,>=0.0.2->dwave-ocean-sdk) (from versions: 4.1.3717, 4.2.3724, 4.2.3758, 4.2.3790, 4.3.3802, 4.3.3803, 4.3.3805, 4.4.3842, 5.0.3919, 5.1.4045)
No matching distribution found for ortools<7.0.0,>=6.6.4659 (from penaltymodel-mip<0.2.0,>=0.1.2; platform_machine != "x86" and python_version != "3.4"->dwavebinarycsp<0.1.0,>=0.0.2->dwave-ocean-sdk)
```
We should determine how far back we can go in ortools

**Comment by randomir** [comment]
Also, we should try to upgrade `ortools` to 9.0+, as that will unblock https://github.com/dwavesystems/dwave-ocean-sdk/issues/144.

**Comment by arcondello** [comment]
As of version [1.0.0](https://github.com/dwavesystems/penaltymodel/releases/tag/1.0.0) we use SciPy rather than ortools so this is no longer relevant.
